keps/sig-node/4815-dra-partitionable-devices/README.md

```diff
@@ -92,9 +101,12 @@ allocated, rather than requiring them to be created before.
 
 ## Motivation
 
-One of the primary motivating examples for supporting partitionable devices
-with DRA is to enable the dynamic allocation of Multi-Instance GPUs
-(MIG) on NVIDIA hardware. MIG devices are represented as fixed-size partitions
+We have two primary motivating examples for supporting partitionable devices
+with DRA. The first is for partitioning a single GPU into smaller partitions, while
+the second is multi-host scheduling of interconnected TPUs.
```

Comment:
For the single-host TPU case, you are planning on a separate PR?

A couple others (see [here](https://docs.google.com/document/d/1lXGfnrBixRIMW9ESa-mv09Kisb2myVFV_A3nqPJ4FCQ/edit?tab=t.0)), which I think are single-host:
- AWS inferentia ring topologies
- AWS trainium 2d torus topologies

For multi-host partitions, @ahg-g may have another use case.

Reply:
There is also Intel's use case for RDMA which @byako and I made sure would work with this design as well (even though it didn't make its way into the KEP).

Reply:
Yeah, so I do want to validate this with other use-cases to make sure it is broadly useful. This doc might be very long if we describe them all in detail here, but it also seems like information that might be useful to inform future decisions, so this might be a good place for it. For initial discussions around it, a doc might be easier though, so I can expand on it https://docs.google.com/document/d/1lXGfnrBixRIMW9ESa-mv09Kisb2myVFV_A3nqPJ4FCQ/edit?usp=sharing

@klueska Are there any docs/examples of the RDMA use-case?

Reply:
I've updated the language here to make it clearer that we are looking at more use-cases, but (at least for now) we are describing two of them in detail.

Reply:
This is the Intel one:
https://github.com/byako/enhancements/commit/199d51362e19a969d6cf71a3b88e62825fda9abe

Reply:
Updated the doc to reference SR-IOV rather than RDMA. Also added a link to the description to https://docs.google.com/document/d/1lXGfnrBixRIMW9ESa-mv09Kisb2myVFV_A3nqPJ4FCQ/edit?usp=sharing